src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -106,73 +109,88 @@ exec newgrp docker
 
 `;
     this.textDataMacOs = `
-#### Part 1a - Install Java dependencies
-We'll cover two ways to install Java 11.
+#### Part 1 - Install Java
```

Comment:
Radical suggestion -- let's not get into into the details of how to install Java, and just have links.

Your steps are very detailed, and yet:

* I didn't install Java on my Mac either of the ways you list; I installed was from https://adoptium.net/, which at the time was the only Apple Silicon native JDK implementation.
* The minor version you have, 17.0.2, is old (the `2` part), I think. While you could update it to the latest, it could probably become quickly out of date.
* There are different ways of switching JVM versions on a Mac -- I use `jenv`, and I don't think we want to be too prescriptive about how to do it. 
* Instructions change with newer releases.

I think it's better not to get too detailed and let the package owners tell you how to install.

What do you think?

Reply:
There is a caveat... if we tell people to use homebrew, they have to install xCode command line tools, and that [can](https://www.reddit.com/r/swift/comments/w7rzfl/command_line_tools_for_xcode_taking_so_long_to/) [take](https://github.com/orgs/Homebrew/discussions/131) [time](https://forums.macrumors.com/threads/if-mbp-m1-max-is-so-fast-why-is-installing-xcode-so-slow.2324058/). If we tell people to use OpenJDK from the archival website, they will be told on said website "hey, this is old software, don't use this!" and will end up with an older version than what homebrew would give them.

That being said: I ultimately agree with this point, and would rather direct people to just use homebrew. Installing homebrew is user-friendly, and iirc it handles the (sometimes iffy) xcode cl tools installation well. People who are developing bioinformatics tools often have it already. Can we investigate the xcode thing on a few different pieces of hardware?

Reply:
I stick by my suggestion that we just not get into how to install Java. :) I wouldn't even recommend homebrew in our doc.

To clarify, I'm not anti-Homebrew; I have it installed (on a Silicon Mac, FWIW) and use it, although I don't happen to use it to install Java. But I don't think it's Dockstore's domain to take a position on how to configure your local machine.

Reply:
Could you cover this change?

Reply:
re-bumping, I can't assess this